opencti-platform/opencti-front/src/private/components/data/Root.jsx

```diff
@@ -42,7 +42,7 @@ const Root = () => {
           path="/ingestion"
           element={
             <Security
-              needs={[SETTINGS]}
+              needs={[SETTINGS, INGESTION]}
```

Comment:
so if I only have SETTINGS capability, I can access ingestion root ? I just tested with a user having only SETTINGS capability, and even though I don't see the "data" menu, I can access the root by url dashboard/data/ingestion/

![image](https://github.com/OpenCTI-Platform/opencti/assets/668192/08617b3b-f8e1-4e8d-903b-ec420c5d4408)

Reply:
oh but maybe this will be done in next chunk ? just to be sure about ingestion capability: is it mandatory to have ingestion capability to access ingestion ?

Reply:
There is no logic in giving SETTINGS access to this route. I wanted to fix it in the next chunk. but it's done, thanks.